<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManufacturersListView</class>
 <widget class="QWidget" name="ManufacturersListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Manufacturers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addManufacturerpushButton">
        <property name="toolTip">
         <string>Add new manufacturer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images.qrc">
          <normaloff>:/images/manufacturers/add_manufacturer.png</normaloff>:/images/manufacturers/add_manufacturer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editManufacturerpushButton">
        <property name="toolTip">
         <string>edit manufacturer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images.qrc">
          <normaloff>:/images/manufacturers/edit_manufacturer.png</normaloff>:/images/manufacturers/edit_manufacturer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeManufacturerpushButton">
        <property name="toolTip">
         <string>remove manufacturer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images.qrc">
          <normaloff>:/images/manufacturers/remove_manufacturer.png</normaloff>:/images/manufacturers/remove_manufacturer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>623</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="manufacturersTableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>CNPJ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>State Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fantasy Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Phone Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fax Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Contact Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Street/Avenue</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Complement</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Square</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>City</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>State</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Postal Code</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
